contracts/MarginBase.sol

```diff
@@ -612,6 +612,10 @@ contract MarginBase is MinimalProxyable, IMarginBase, OpsReady {
         onlyOwner
         returns (uint256)
     {
+        require(
+            address(this).balance >= 1 ether / 10,
+            "Min 0.1 ETH balance in account"
+        );
```

Comment:
Let me know if you think this is better with a custom error

Reply:
I think it would be good to have a custom error here

Reply:
could also use `100 finney` or `1e17 wei` to avoid division which costs gas

Reply:
I believe there's compiler optimization step that evaluate these expressions and turns them into constant values, but will make the update for the custom error